x-pack/plugins/xpack_main/public/hacks/check_xpack_info_change.js

```diff
@@ -20,30 +22,34 @@ module.factory('checkXPackInfoChange', ($q, Private) => {
   const xpackInfoSignature = Private(XPackInfoSignatureProvider);
   const debounce = Private(DebounceProvider);
   const isLoginOrLogout = Private(PathProvider).isLoginOrLogout();
+  let licenseExpirationBannerId;
```

Comment:
When I say that the banner id was being tracked I assumed it was so we could clear the banner later, which doesn't seem to be the case. Perhaps a boolean that tracks if it was shown, `licenseExpirationBannerShown` maybe, would be more descriptive?